### PR TITLE
Draft of release highlights

### PR DESCRIPTION
**Pull request by desertaxle**
### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
 
-- Adds work pool creation wizard — https://github.com/PrefectHQ/prefect/pull/9717
-- Add interactive prompts for deployment name and work pool to `prefect deploy` — https://github.com/PrefectHQ/prefect/pull/9707
-- Add interactive prompt to `prefect work-pool create` — https://github.com/PrefectHQ/prefect/pull/9686
+### Interactive Deployments and Work Pool Wizard 🧙
+![Screenshot 2023-05-25 at 1 02 48 PM](https://github.com/PrefectHQ/prefect/assets/12350579/c861b8dd-2dbb-4cfa-82f9-69008714f9fe)
+
+This release introduces enhancements aimed at simplifying deployment and work pool creation.
+
+Firstly, the `prefect deploy` command has been upgraded to provide interactive prompts for deployment names and work pool selections. If you don't provide a deployment name via the CLI or a deployment.yaml file, Prefect will prompt you to do so. Furthermore, if a work pool name isn't specified, Prefect will guide you through the available work pools for your workspace. This feature aims to make deployments more approachable, even for first-time users, requiring just an entrypoint to a flow to get started.
```
Comment (serinamarie) [inline RELEASE-NOTES.md:10]:
```suggestion
Firstly, the `prefect deploy` command has been upgraded to provide interactive prompts for deployment names and work pool selections. If you don't provide a deployment name via the CLI or a `deployment.yaml` file, Prefect will prompt you to do so. Furthermore, if a work pool name isn't specified, Prefect will guide you through the available work pools for your workspace. This feature aims to make deployments more approachable, even for first-time users, requiring just an entrypoint to a flow to get started.
```

I'm unsure if it makes more sense with our release notes cadence to say "you will be prompted/guided" versus "Prefect will prompt/guide you". @billpalombi would know better than I

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
+
+Secondly, we've added a work pool creation wizard to streamline the process and spotlight various infrastructure types. The wizard will walk you through the essentials: basic work pool info, infrastructure type, and infrastructure configuration. The infrastructure type step will present you with a list of available infrastructure types, each with an icon and a description.
+
```
Comment (serinamarie) [inline RELEASE-NOTES.md:13]: A UI screenshot or two here would be super

File: RELEASE-NOTES.md (hunk 1 above)
Comment (desertaxle) [reply RELEASE-NOTES.md]: I changed from 'Prefect' to 'the CLI.' Let me know if you think that reads better.

File: RELEASE-NOTES.md (hunk 2 above)
Comment (desertaxle) [reply RELEASE-NOTES.md:16]: Added!

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,10 +1,25 @@
...
+## Release 2.10.11
 
-- Adds work pool creation wizard — https://github.com/PrefectHQ/prefect/pull/9717
-- Add interactive prompts for deployment name and work pool to `prefect deploy` — https://github.com/PrefectHQ/prefect/pull/9707
-- Add interactive prompt to `prefect work-pool create` — https://github.com/PrefectHQ/prefect/pull/9686
+### Interactive Deployments and Work Pool Wizard 🧙
+
+This release introduces enhancements aimed at simplifying deployment and work pool creation.
+
+![Screenshot 2023-05-25 at 1 02 48 PM](https://github.com/PrefectHQ/prefect/assets/12350579/c861b8dd-2dbb-4cfa-82f9-69008714f9fe)
```
Comment (madkinsz) [inline RELEASE-NOTES.md:9]: Make sure to update this alt-text to something relevant

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,10 +1,25 @@
...
 
-## Release preview
+## Release 2.10.11
 
-- Adds work pool creation wizard — https://github.com/PrefectHQ/prefect/pull/9717
-- Add interactive prompts for deployment name and work pool to `prefect deploy` — https://github.com/PrefectHQ/prefect/pull/9707
-- Add interactive prompt to `prefect work-pool create` — https://github.com/PrefectHQ/prefect/pull/9686
+### Interactive Deployments and Work Pool Wizard 🧙
+
+This release introduces enhancements aimed at simplifying deployment and work pool creation.
```
Comment (billpalombi) [inline RELEASE-NOTES.md:7]:
```suggestion
This release simplifies deployment and work pool creation.
```

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,10 +1,25 @@
...
+
+![interactive-prefect-deploy-console-output](https://github.com/PrefectHQ/prefect/assets/12350579/c861b8dd-2dbb-4cfa-82f9-69008714f9fe)
+
+Firstly, the `prefect deploy` command has been upgraded to provide interactive prompts for deployment names and work pool selections. If you don't provide a deployment name via the CLI or a `deployment.yaml` file, the CLI will prompt you to do so. Furthermore, if a work pool name isn't specified, the CLI will guide you through the available work pools for your workspace. This feature aims to make deployments more approachable, even for first-time users, requiring just an entrypoint to a flow to get started.
```
Comment (billpalombi) [inline RELEASE-NOTES.md:11]:
```suggestion
Firstly, the `prefect deploy` command has been upgraded to provide interactive prompts for deployment names and work pool selections. If you don't provide a deployment name via the CLI or a `deployment.yaml` file, the CLI will prompt you to do so. Furthermore, if a work pool name isn't specified, the CLI will guide you through the available work pools for your workspace. This feature aims to make deployments more approachable, especially for first-time users, requiring just an entrypoint to a flow to get started.
```

File: RELEASE-NOTES.md (hunk 1 above)
Comment (billpalombi) [reply RELEASE-NOTES.md]: We have not been consistent on active vs passive voice. This reads well to me.

**Review by serinamarie — APPROVED** [review]

**Review by billpalombi — APPROVED** [review]
LGTM

**Pull request merged**